script.js

```diff
@@ -0,0 +1,132 @@
+
+import { updateTime } from './Modules/clock.js';
+updateTime ();
+import * as Navbar from './Modules/nav-bar.js'
+
+const bookList = document.getElementById('book-list');
+const addBook = document.getElementById('add-book');
+const newTitle = document.getElementById('new-title');
+const newAuthor = document.getElementById('new-author');
+const allBooks = document.getElementById('all-books');
+let bookData = [];
+
+function setBorder() {
+  const data = JSON.parse(localStorage.getItem('bookData'));
+  if (bookData.length > 0 || data.length > 0) {
+    allBooks.classList.add('border');
+  } else if (bookData.length === 0 && data.length === 0) {
+    allBooks.classList.remove('border');
+  }
+}
+function setStyles() {
```

Comment:
- [ ] Kindly modify this to use ES6 syntax by changing it to `arrow function` 👍🏽